### ISSUE TITLE
Carousel: poor image quality

### ISSUE THREAD

**Issue by eliorivero** [Automattic/jetpack#14113]
I'm using Tiled Gallery and Carousel to display some images. Photon CDN is enabled for images.
While the images in Tiled Gallery are fine, the image shown in Carousel is blurry and low quality.

Compare for example the blurry image used by the slideshow:
https://i0.wp.com/elio.blog/wp-content/uploads/2019/11/5-free-capture-one-styles-5.jpg?zoom=2&fit=1044%2C557

with the good one used in the Tiled Gallery in the content:
https://i1.wp.com/elio.blog/wp-content/uploads/2019/11/5-free-capture-one-styles-5.jpg?ssl=1

#### Steps to reproduce the issue

1. Go to https://elio.blog/5-free-styles-for-capture-one/
2. right click on one of the images, open in a new tab, and check its good quality
3. back to the post. Click on one of the images to launch the Carousel, check its poor quality

#### What I expected
The image in the Carousel to have good quality, like the one in Tiled Gallery.

#### What happened instead
The image in the Carousel has low quality and almost look blurry.

**Comment by jeherve** [comment]
Yeah, I can definitely see a difference. Do things look much better when you disable Photon?

**Comment by eliorivero** [comment]
Yes, of course. It's absolutely better when I disable Photon because it uses the images I uploaded https://elio.blog/wp-content/uploads/2019/11/5-free-capture-one-styles-5.jpg
Not an option because I lose the Tiled Gallery.

**Comment by jeherve** [comment]
Cool, thanks for confirming. The good news is, until this gets solved you can disable Photon; the Tiled Gallery block will remain available when you do.

It may actually be worth going to Jetpack > Beta first, and switching between Bleeding Edge and RC, and then back to Bleeding Edge to make sure you're running the most up to date version of the plugin right now. Let's see if this changes anything with the markup of the gallery, since we've made a few changes to all this in the past few months while your site's Jetpack plugin stayed stuck to 7.4.

**Comment by eliorivero** [comment]
No improvement at all. I've now updated the bleeding edge I was using.

<img width="739" alt="Screen Shot 2019-11-26 at 09 28 32" src="https://user-images.githubusercontent.com/1041600/69633585-4546da00-102f-11ea-85bd-5a2404d4024f.png">

**Comment by jeherve** [comment]
cc @darthhexx; do you think you could look at this? Why would the resized image be so blurry compared to the full-size one?
https://i0.wp.com/elio.blog/wp-content/uploads/2019/11/5-free-capture-one-styles-5.jpg?zoom=2&fit=1044%2C557

**Comment by dres-de** [comment]
Hi there,

I have another example of this issue in case it helps look into it:

https://enquepiensauncalcetin.com/2018/08/sri-lanka-kandy/ (there are two galleries in the post).

This link should open the image in the slideshow: https://enquepiensauncalcetin.com/2018/08/sri-lanka-kandy/#jp-carousel-5788

You can see that the i1.wp.com URL that is used as the source of the image (https://i1.wp.com/enquepiensauncalcetin.com/wp-content/uploads/2018/08/peradeniya_03.jpg**?w=688&h=458&ssl=1**) is a size that, when the carousel is in full screen on a laptop, makes the image low quality. You can still click on the “see full size” link and get the higher quality image: https://i1.wp.com/enquepiensauncalcetin.com/wp-content/uploads/2018/08/peradeniya_03.jpg

**Comment by sarangsss29** [comment]
Hi,
I also have the same issue on my website:
So I have enabled Jetpack Photon or  Site accelerator / and the images are so poor quality that users can't even read the text on them and distinguish what's on them. When I upload the images they are high quality and already optimized/compressed for viewing. But the Jetpack compresses them so much that one can't even see it. What's the point of the Photon if the user can't even see the text on images properly?

Here is an example of the images in a gallery or caurosel on this page
https://www.androidsage.com/2020/08/17/oxygen-os-11-enables-dual-sim-5g-for-both-slots/

https://app.box.com/s/zfm986y5ncoh5j0h7gr5n2ep9zitm1gc
https://app.box.com/s/xm7s4mbuebgioll7m6o4fxjxyhrmit19

Looks quite low quality, especially on mobile. It is also unreadable on desktop. 

And look at the quality I uploaded!
https://www.androidsage.com/wp-content/uploads/2020/08/OnePlus-8-Pro-dual-sim-5g-feature-with-oxygen-os-11.jpg

**Comment by sarangsss29** [comment]
Hi @jeherve,
Any fixes yet? I also tried other lightbox plugins and fancybox plugins for images and galleries, they do not work. So we are stuck with jetpack, but it's quite poor quality.

**Comment by htdat** [comment]
FYI @sarangsss29 is contacting us via 3270522-zen too.

**Comment by phillipjohnston** [comment]
Carousel image quality is still much poorer than the image quality in other blocks...

**Comment by sarangsss29** [comment]
Can you bring an update to this?

**Comment by sdixon194** [comment]
Internally reported here: p8oabR-FS-p2#comment-5324
